analyzer/src/main/kotlin/curation/OrtConfigPackageCurationProvider.kt

```diff
@@ -61,10 +61,10 @@ open class OrtConfigPackageCurationProvider : PackageCurationProvider {
         }
     }
 
-    override fun getCurationsFor(packages: Collection<Package>): Map<Identifier, List<PackageCuration>> =
-        packages.mapNotNull { pkg ->
-            getCurationsFor(pkg.id).takeUnless { it.isEmpty() }?.let { pkg.id to it }
-        }.toMap()
+    override fun getCurationsFor(packages: Collection<Package>): List<PackageCuration> =
+        packages.flatMap { pkg ->
+            getCurationsFor(pkg.id)
+        }.distinct()
```

Comment:
Nit: This could be come a one-liner now to align with `Sw360PackageCurationProvider`.